webapp/web/configuration.css

```
@@ -230,6 +249,34 @@
   cursor: pointer;
 }
 
+.remove-conversation-responses-modal {
+  display: grid;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.7);
```

I think this would look better lighter in colour:
```suggestion
  background: rgba(0, 0, 0, 0.33);
```